controllers/openstacknet_controller.go

```diff
@@ -73,5 +158,7 @@ func (r *OpenStackNetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 func (r *OpenStackNetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&ospdirectorv1beta1.OpenStackNet{}).
+		// TODO: check for why there are unwanted/not required reconcile events
+		//Owns(&networkv1.NetworkAttachmentDefinition{}).
```

Comment:
This might be caused by line 74's cluster-scoped NAD perms.  Without a watch filter function, it could be that any change to any NAD in any namespace will wake the controller's `Reconcile` loop.

Reply:
good point, thanks